gtsam/CMakeLists.txt

```diff
@@ -101,6 +101,7 @@ message(STATUS "Building GTSAM - shared: ${BUILD_SHARED_LIBS}")
 
 # BUILD_SHARED_LIBS automatically defines static/shared libs:
 add_library(gtsam ${gtsam_srcs})
+target_compile_features(gtsam PUBLIC cxx_std_11)
```

Comment:
The `cxx_std_11` feature was introduced in [CMake 3.8](https://cmake.org/cmake/help/v3.8/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html). Ubuntu 16.04 has CMake 3.5.1 by default, so unfortunately we can't use this.
Looks like Travis has CMake 3.12, but the property still wasn't set. Don't know why.

Reply:
You're right! Let's see if https://github.com/borglab/gtsam/pull/27/commits/53cf33b045426bd1b443c0297d1d6f9d062a70c8 makes it...